src/mahoji/commands/mine.ts

```diff
@@ -15,36 +15,173 @@ import { mahojiUsersSettingsFetch } from '../mahojiSettings';
 const pickaxes = [
 	{
 		id: itemID('Crystal pickaxe'),
-		reductionPercent: 11,
+		ticksBetweenRolls: 2.75,
 		miningLvl: 71
 	},
 	{
 		id: itemID('Infernal pickaxe'),
-		reductionPercent: 6,
+		ticksBetweenRolls: 2.83,
 		miningLvl: 61
 	},
 	{
 		id: itemID('Dragon pickaxe'),
-		reductionPercent: 6,
+		ticksBetweenRolls: 2.83,
 		miningLvl: 61
+	},
+	{
+		id: itemID('Rune pickaxe'),
+		ticksBetweenRolls: 3,
+		miningLvl: 41
+	},
+	{
+		id: itemID('Adamant pickaxe'),
+		ticksBetweenRolls: 4,
+		miningLvl: 31
+	},
+	{
+		id: itemID('Mithril pickaxe'),
+		ticksBetweenRolls: 5,
+		miningLvl: 21
+	},
+	{
+		id: itemID('Black pickaxe'),
+		ticksBetweenRolls: 5,
+		miningLvl: 11
+	},
+	{
+		id: itemID('Steel pickaxe'),
+		ticksBetweenRolls: 6,
+		miningLvl: 6
+	},
+	{
+		id: itemID('Iron pickaxe'),
+		ticksBetweenRolls: 7,
+		miningLvl: 1
+	},
+	{
+		id: itemID('Bronze pickaxe'),
+		ticksBetweenRolls: 8,
+		miningLvl: 1
 	}
 ];
 
 const gloves = [
 	{
 		id: itemID('Expert mining gloves'),
-		reductionPercent: 6
+		'Silver ore': 50,
+		Coal: 40,
+		'Gold ore': 33.33,
+		'Mithril ore': 25,
+		'Adamantite ore': 16.66,
+		'Runite ore': 12.5,
+		Amethyst: 25
 	},
 	{
 		id: itemID('Superior mining gloves'),
-		reductionPercent: 4
+		'Silver ore': 0,
+		Coal: 0,
+		'Gold ore': 0,
+		'Mithril ore': 25,
+		'Adamantite ore': 16.66,
+		'Runite ore': 12.5,
+		Amethyst: 0
 	},
 	{
 		id: itemID('Mining gloves'),
-		reductionPercent: 2
+		'Silver ore': 50,
+		Coal: 40,
+		'Gold ore': 33.33,
+		'Mithril ore': 0,
+		'Adamantite ore': 0,
+		'Runite ore': 0,
+		Amethyst: 0
+	}
+];
+
+const varrockArmours = [
+	{
+		id: itemID('Varrock armour 4'),
+		Clay: 10,
+		'Copper ore': 10,
+		'Tin ore': 10,
+		'Iron ore': 10,
+		'Silver ore': 10,
+		Coal: 10,
+		Sandstone: 10,
+		'Gold ore': 10,
+		Granite: 10,
+		'Mithril ore': 10,
+		'Adamantite ore': 10,
+		'Runite ore': 10,
+		Amethyst: 10
+	},
+	{
+		id: itemID('Varrock armour 3'),
+		Clay: 10,
+		'Copper ore': 10,
+		'Tin ore': 10,
+		'Iron ore': 10,
+		'Silver ore': 10,
+		Coal: 10,
+		Sandstone: 10,
+		'Gold ore': 10,
+		Granite: 10,
+		'Mithril ore': 10,
+		'Adamantite ore': 10,
+		'Runite ore': 0,
+		Amethyst: 0
+	},
+	{
+		id: itemID('Varrock armour 2'),
+		Clay: 10,
+		'Copper ore': 10,
+		'Tin ore': 10,
+		'Iron ore': 10,
+		Silver: 10,
+		Coal: 10,
+		Sandstone: 10,
+		'Gold ore': 10,
+		Granite: 10,
+		'Mithril ore': 10,
+		'Adamantite ore': 0,
+		'Runite ore': 0,
+		Amethyst: 0
+	},
+	{
+		id: itemID('Varrock armour 1'),
+		Clay: 10,
+		'Copper ore': 10,
+		'Tin ore': 10,
+		'Iron ore': 10,
+		'Silver ore': 10,
+		Coal: 10,
+		Sandstone: 0,
+		'Gold ore': 0,
+		Granite: 0,
+		'Mithril ore': 0,
+		'Adamantite ore': 0,
+		'Runite ore': 0,
+		Amethyst: 0
 	}
 ];
 
+const miningCapeOreEffect = {
+	id: itemID('Mining cape'),
+	Clay: 5,
+	'Copper ore': 5,
+	'Tin ore': 5,
+	'Iron ore': 5,
+	'Silver ore': 5,
```

Comment:
this object makes no sense, or atleast the prescence of `id` - shouldnt mix unrelated/random things together.  You can use a `Bank` for this.